### PR TITLE
[rnbw-balance] Add rnbw-balance strategy

### PR DESCRIPTION
**Pull request by mystbrent**
Fixes # .

Changes proposed in this pull request:
- Added rnbw-balance strategy for HaloDAO protocol

### CODE REVIEW

**Comment by ChaituVR** [comment]
@mystbrent Make sure to add your strategy to `/src/strategies/index.ts` file, Thanks.

File: src/strategies/rnbw-balance/index.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import { formatUnits } from '@ethersproject/units';
+import Multicaller from '../../utils/multicaller';
+
+export const author = 'amaterasu';
```
Comment (ChaituVR) [inline src/strategies/rnbw-balance/index.ts:4]: this should be your github username

Comment (mystbrent) [reply src/strategies/rnbw-balance/index.ts]: Thank you for reviewing @ChaituVR. The issues are resolved in the latest pushed commit.

**Comment by ChaituVR** [comment]
@mystbrent would be great if you can resolve the conflicts

**Comment by mystbrent** [comment]
> @mystbrent would be great if you can resolve the conflicts

Resolved on the latest push. Sorry did not realize there were new commits

File: src/strategies/rnbw-balance/index.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+
+  multi.call('dsrtPrice', options.token, 'getCurrentHaloHaloPrice');
+
+  addresses.forEeach((address) => {
```
Comment (ChaituVR) [inline src/strategies/rnbw-balance/index.ts:52]: Hi @mystbrent Sorry for not mentioning all issue at once, here there is an issue it is not `forEach`? 
Make sure to:
- Update the branch
- Allow Maintainers to update the branch
- Make sure everything is running without any error when you run `npm test --strategy=rnbw-balance`

Thanks

Comment (mystbrent) [reply src/strategies/rnbw-balance/index.ts:52]: Hi @ChaituVR thanks for reviewing. 

It seems that there is an issue with the testing this strategy when the network is in Rinkeby?

Are there any test networks that can be used for the test? 

Please let me know.

Thank you

Comment (mystbrent) [reply src/strategies/rnbw-balance/index.ts]: Issue has been resolved on the latest pushed commit

**Comment by mystbrent** [comment]
Issue has been resolved on the latest pushed commit

File: src/strategies/rnbw-balance/index.ts (hunk 2 above)
Comment (ChaituVR) [reply src/strategies/rnbw-balance/index.ts]: @mystbrent Please Allow Maintainers to update the branch https://docs.github.com/assets/images/help/pull_requests/allow-maintainers-to-make-edits-sidebar-checkbox.png

Comment (mystbrent) [reply src/strategies/rnbw-balance/index.ts]: Hi @ChaituVR, sorry for the delay the Allow maintainers to update checkbox is not visible from my end. Did some research and found this article:

https://github.community/t/how-can-we-enable-allow-edits-from-maintainers-by-default/2847/5.

I am still looking for a workaround with this.

Thank you.

**Review by bonustrack — APPROVED** [review]
Looks good!

**Pull request merged**